scripts/tests/test_zkai_attention_kv_risc0_semantics_receipt_gate.py

```
@@ -0,0 +1,191 @@
+if SPEC is None or SPEC.loader is None:
+    raise RuntimeError(f"failed to load attention/KV RISC Zero receipt gate from {SCRIPT_PATH}")
+GATE = importlib.util.module_from_spec(SPEC)
+sys.modules[SPEC.name] = GATE
+SPEC.loader.exec_module(GATE)
+
+
+class ZkAiAttentionKvRisc0SemanticsReceiptGateTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        if not GATE.JSON_OUT.exists():
+            raise unittest.SkipTest("attention/KV RISC Zero semantics receipt evidence has not been generated yet")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)